### PR TITLE
Navigations

### PR DESCRIPTION
**Pull request by Abhi-0987**

### CODE REVIEW

**Comment by maheshalyana** [comment]
What about viewmore button

File: lib/screens/home_screen_tab_structure.dart [hunk 1]
```diff
@@ -181,6 +194,9 @@ class RequestTile extends StatefulWidget {
 class _RequestTileState extends State<RequestTile> {
   @override
   Widget build(BuildContext context) {
-    return RideDetailsTile(values: widget.values);
+    return RideDetailsTile(
+      values: widget.values,
+      path: 'one',
```
Comment (maheshalyana) [inline lib/screens/home_screen_tab_structure.dart:199]: instead send the class directly `path: ViewRequestsScreen()`

**Review by maheshalyana — CHANGES_REQUESTED** [review]

File: lib/screens/home_screen_tab_structure.dart (hunk 1 above)
Comment (Abhi-0987) [reply lib/screens/home_screen_tab_structure.dart:199]: I am unable to pass the route like that. I tried doing that first. I tried to send class directly by keeping the datatype as dynamic. But there was an error in the `MaterialPageRoute` it was giving error if I use that path there. Then at last I did this.

Comment (maheshalyana) [reply lib/screens/home_screen_tab_structure.dart:199]: Okay

**Review by maheshalyana — APPROVED** [review]

**Review by tanishq-2208 — APPROVED** [review]
https://github.com/NewTab-Technnologies/Ride-Bhaiya/assets/152472135/253b466b-9e19-43b5-bb9c-3334bbfd2696

Working As Expected.

**Pull request merged**